waltz-schema/src/main/resources/liquibase/db.changelog-1.67.xml

```diff
@@ -24,4 +24,26 @@
                author="patidev">
         <tagDatabase tag="v1.66"/> <!-- tag the current db version, i.e. the last  -->
     </changeSet>
+
+    <changeSet id="20250123"
```

Comment:
the change set id's should have the form:

<date>-<issue>-<step>

where:

- date is yyyymmdd
- issue is the _waltz_ issue number (not your orgs), e.g. 7202
- step is a counter in case a single change is split across multiple steps, e.g. 1, 2

so this changeset id should be `20240123-7202-1`

The README.md in this directory (https://github.com/finos/waltz/blob/master/waltz-schema/src/main/resources/liquibase/README.md) has a more detailed explanation